src/fetch/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@kiwicom/fetch",
+  "name": "@adeira/fetch",
   "private": false,
   "license": "MIT",
   "version": "2.5.1",
```

Comment:
We should reset the version and just backup the changelog like I did here: https://github.com/adeira/universe/blob/master/src/eslint-config-kiwicom/CHANGELOG.md (no new changelog entry - it's new package)

Reply:
Sure, I was thinking about that as well, I will just set it to `1.0.0` then